### PR TITLE
Fix/4732 scrolling glitch

### PR DESCRIPTION
**Pull request by nickguendling**
## Description
Fixes scrolling glitches and performance issues on the home screen.

## Links to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4732
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4776

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift [hunk 1]
```diff
@@ -72,17 +72,6 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 			}
 			.store(in: &subscriptions)
 
-		viewModel.state.$statistics
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:75]: Is there another place where the statistics are updated?

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift [hunk 2]
```diff
@@ -363,6 +361,10 @@ class HomeTableViewController: UITableViewController, NavigationBarOpacityDelega
 	}
 
 	private func riskCell(forRowAt indexPath: IndexPath) -> UITableViewCell {
+		if let riskCell = riskCell {
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:364]: Wouldn't this prevent the risk-cell to update with a new model and a new risk-state?

**Review by ArturFriesen — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift (hunk 1 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:75]: The statistics data is updated in the HomeState which triggers the update directly inside the statistics cell, there's no need to update the table view around it. This was necessary when we did not show the cell at all if the statistics data was not there yet. But we later changed it to initially show the cell with a height of 0 and then let it update itself internally, so I removed this code.

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift (hunk 2 above)
Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/Home/HomeTableViewController.swift:364]: The risk cell is also updated from the HomeState, that's why we only need to set it up once.

**Review by ArturFriesen — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Pull request merged**